GLOSSARY.md

```diff
@@ -0,0 +1,75 @@
+# Glossary
+
+## Design Studio
+
+How we refer to the entire application and all its subsystems. You may often see us refer to "modeling app", "the app", or "ZDS", which are all synonymous.
+
+## Top Bar
+
+The interface element found at the top (yay descriptive names!) of our layouts. It is used for navigating around the app, managing your user and project information, and sharing.
+
+## Status Bar
+
+The interface element found at the bottom of our layouts. It is used to get information about where you are and what you're doing in the application at the moment.
+
+The left side is for "global" items like your network status, and the right side is for "local", context-specific information such as your current selection in the Workbench.
+
+## Home
+
+The interface within Design Studio where users can choose which project to open, and can create new ones. It consists of a Top Bar, Project Browser, and Status Bar.
+
+![Home](/public/glossary/home.png)
+
+## Project Browser
+
+The interface element found in the middle of the Home interface. It shows projects available to be opened in an area on the right, and a sidebar on the left with action buttons to create blank, generated, or sample projects.
+
+## Workbench
```

Comment:
As mentioned in the FE sync call, I'm concerned we're overloading this term iff it's exposed in the frontend UI, docs, or other public facing stuff. We might want to reserve this term for the modal change to CAD/CAM/Sheet Metal/Surface workbenches/workspaces when we introduce that functionality. 

We could call it a work _space_ and use work _bench_ for the modal change, or maybe we call it a "mode"? Like "CAM Mode", although I don't like this option very much.

To be clear, this is inspired by Fusion's configuration and naming which uses "workspace": 

<img width="526" height="606" alt="Image" src="https://github.com/user-attachments/assets/05f76703-c249-4608-972f-93a4a96692c6" />

Reply:
Okay I think I'd like to call the UI in its totality the Workspace, and call it the Manufacture Workbench. I know that breaks with the example from the industry here, but it my ears the metaphor works better: a workspace contains many workbenches. What do you think?

Reply:
"Workspace" probably won't show up in the UI, it's just something we can all point to as developers and while helping users to refer to "that thing over there where you do stuff"